rewrite in rust

Fusselwurm opened the pull request:
* rewrite in Rust
* added x64 builds for Linux
* a bit of refactoring in sqf code

Fusselwurm committed:
remove C++ code

Fusselwurm committed:
add Rust version

Fusselwurm committed:
remove more c++ compilation stuff

Commit:
hmm build action

Commit:
fix release actions

jokoho48 commented:
> Nobody's running arma32 anymore, not even linux dedi. :P  

there are a few users that have AMD FX processor and using a Logitech USB Headset that are forced to use 32bit due to a bug that kills framerates when inside vehicles.

Commit:
trying my hand at 32bit builds

Commit:
use new callExtension signature

Fusselwurm committed:
adjust *release* action

Fusselwurm committed:
adjust extension client code to new callExtension syntax

Fusselwurm committed:
shrink gitignore

shrink to whats relevant for this project. everything else belongs into your global gitignore

Fusselwurm committed:
add missing config for win32
